hack/verify-codecgen.sh

```diff
@@ -86,12 +86,16 @@ index=(${result})
 # Build codecgen from Godeps.
 # However, we need to install godep first.
 # We make some tricks with GOPATH variable to make it work with Travis.
-_gopath=${GOPATH}
-export GOPATH="${_tmpdir}"
-go get -u github.com/tools/godep 2>/dev/null
-go install github.com/tools/godep 2>/dev/null
-GODEP="${_tmpdir}/bin/godep"
-export GOPATH=${_gopath}
+if [[ -n "${TRAVIS:-}" ]]; then
```

Comment:
I was making it have parity with update- when outside of travis.  Should we
change them both?  In general we probably need to get a handle about how
scripts should do this so we don't wind up with a ton of different ways.
Does it need to be a util::install_godep method?

On Tue, Dec 1, 2015 at 3:43 PM, Jeff Grafton <notifications@github.com>
wrote:

> In hack/verify-codecgen.sh
> <https://github.com/kubernetes/kubernetes/pull/17960#discussion_r46335961>
> :
>
> > @@ -86,12 +86,16 @@ index=(${result})
> >  # Build codecgen from Godeps.
> >  # However, we need to install godep first.
> >  # We make some tricks with GOPATH variable to make it work with Travis.
> > -_gopath=${GOPATH}
> > -export GOPATH="${_tmpdir}"
> > -go get -u github.com/tools/godep 2>/dev/null
> > -go install github.com/tools/godep 2>/dev/null
> > -GODEP="${_tmpdir}/bin/godep"
> > -export GOPATH=${_gopath}
> > +if [[ -n "${TRAVIS:-}" ]]; then
>
> or maybe just
>
> if ! command -v godep &>/dev/null; then
>
> so this installs godep if necessary, otherwise uses the installed version?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/17960/files#r46335961>.
>